performance/src/performance/timing_info.cpp

```diff
@@ -19,9 +19,10 @@ namespace traccc::performance {
 std::chrono::nanoseconds timing_info::get_time(
     std::string_view timer_name) const {
 
-    auto it = std::find_if(
-        data.begin(), data.end(),
-        [&timer_name](timing_info_pair it) { return it.first == timer_name; });
+    auto it = std::find_if(data.begin(), data.end(),
+                           [&timer_name](timing_info_pair itr) {
+                               return itr.first == timer_name;
+                           });
```

Comment:
Not a problem with this PR per se, but why isn't this just `std::find`? 🤨

Reply:
I don't understand. :frowning: How should one do this with [std::find](https://en.cppreference.com/w/cpp/algorithm/find)? The elements of `data` are pairs. We want to find the first element whose `first` variable/element is equal to `timer_name`.

I'm pretty sure that `std::find_if` **is** the way to do this in the most compact way. :thinking: Or are you proposing changing the type of `traccc::performance::timing_info_pair` to `std::map<std::string, std::chrono::nanoseconds>`? :thinking: That would probably not be a bad idea indeed. (Since we don't allow multiple uses of the same "name" anyway.)

But that I would not want to do in this PR.